graphein/protein/features/nodes/amino_acid.py

```diff
@@ -248,3 +249,46 @@ def hydrogen_bond_acceptor(
     if not sum_features:
         features = np.array(features > 0).astype(int)
     d["hbond_acceptors"] = features
+
+
+
+"""
+TODO: add a similar 'load in' function from .csv as above?
+or stick to hydrophobicity as dict?
+
+TODO: make vector of all hydrophobicity scales instead of one chosen scale?
```

Comment:
I don't think this is necessary. One could refactor the hydrophobicity func to add the attribute as e.g. `hydrophobicity_kd` rather than `hydrophobicity` in case users want to add multiple hydrophobicity scales. I don't have particularly strong feelings about this, so leave it to you.